webapp/portlet/src/main/webapp/groovy/social/webui/UISocialPortalApplicationHead.gtmpl

```diff
@@ -61,4 +61,5 @@
 
      eXo.env.portal.SpaceHomeLayoutResetEnabled = <%=featureService.isFeatureActiveForUser("SpaceHomeLayoutReset", userName)%>;
      eXo.env.portal.PublisherRolePromotionFeatureEnabled = <%=featureService.isFeatureActiveForUser("publisherRolePromotion", userName)%>;
+     eXo.env.portal.NewEditNavigationDrawerEnabled = <%=featureService.isFeatureActiveForUser("newEditNavigationDrawer", userName)%>;
```

Comment:
to not add an eXo addon feature Flag inside Social, you can declare it in the gtmpl of layout-management addon.
@rdenarie FYI.